packages/strapi-connector-mongoose/lib/queries.js

```diff
@@ -435,20 +448,23 @@ module.exports = ({ model, strapi }) => {
     }
 
     // Create entry with no-relational data.
-    const entry = await model.create(data);
-
+    // When specify options, data must be an array.
+    const [entry] = await model.create([data], { session });
```

Comment:
Can you clarify the error message to mention it is a mongoose requirement to call the method with an array if we pass a second argument to create so we get more context on this strange behavior :)

Reply:
Make sense. I added extra explanation.